### PR TITLE
highlight freeloaded shifts in user myshifts view

### LINKED ISSUE #1278: Highlight freeloaded shifts
It would be great to see which shifts got freeloaded by adding a red border, as done for current shifts in #1275

### PR DESCRIPTION
**Pull request by xuwhite**
Screenshot 2023-12-17 at 13 38 14

closes #1278

### CODE REVIEW

File: includes/view/User_view.php [hunk 1]
```diff
@@ -412,14 +413,18 @@ function User_view_myshifts(
...
-                $shift['row-class'] = 'border border-info border-2';
-            } elseif ($after && Carbon::now() > $shift['end'] && Carbon::now() < $after['start']) {
-                $shift['row-class'] = 'border-bottom border-info';
-            } elseif (!$before && Carbon::now() < $shift['start']) {
-                $shift['row-class'] = 'border-top-info';
-            } elseif (!$after && Carbon::now() > $shift['end']) {
-                $shift['row-class'] = 'border-bottom border-info';
+            if ($shift['freeloaded']) {
+                $shift['row-class'] = 'border border-danger border-2';
+            } else {
```
Comment (MyIgel) [inline includes/view/User_view.php:418]: @xuwhite that could be just another elseif here?

File: includes/view/User_view.php [hunk 2]
```diff
@@ -412,14 +413,18 @@ function User_view_myshifts(
...
+                if (Carbon::now() > $shift['start'] &&  Carbon::now() < $shift['end']) {
+                    $shift['row-class'] = 'border border-info border-2';
+                } elseif ($after && Carbon::now() > $shift['end'] && Carbon::now() < $after['start']) {
+                        $shift['row-class'] = 'border-bottom border-info';
```
Comment (MyIgel) [inline includes/view/User_view.php:422]: & that is one indentation too much ;)

File: includes/view/User_view.php (hunk 1 above)
Comment (xuwhite) [reply includes/view/User_view.php:418]: nope that is an else on purpose

File: includes/view/User_view.php (hunk 2 above)
Comment (xuwhite) [reply includes/view/User_view.php:422]: done

File: includes/view/User_view.php (hunk 1 above)
Comment (MyIgel) [reply includes/view/User_view.php]: But what?

**Pull request merged**